apps/admin/package.json

```diff
@@ -19,7 +19,6 @@
     "@repo/auth": "workspace:*",
     "@repo/db": "workspace:*",
     "@repo/emails": "workspace:*",
-    "@repo/lint": "workspace:*",
```

Comment:
we dont need these?

Reply:
Dog I don't know im just trying to get this to pass and I'd allow satan himself access to my code editor for it.

Reply:
OK IDK why this is still showing up, it's still in there so i think that's a bug?